CONTRIBUTING.md

```diff
@@ -0,0 +1,39 @@
+# Welcome to 42 norminette contributing guide
+
+We appreciate your interest in contributing to our project. <br />
+Please take a moment to read through this guide to understand how you can contribute effectively.
+
+## Issue Tracker
+
+Before starting your contribution, we recommend checking our issue tracker [here](https://github.com/42School/norminette/issues). <br />
+It lists the current tasks, bug reports, and feature requests.
+If you find an open issue that you'd like to work on, please comment on it to let us know.
+
+If you encounter a new issue or have a feature request that is not already listed, please open a new issue following the given template.
+
+## Getting Started
+
+To contribute to this project, follow these steps:
+
+1. Fork the repository to your GitHub account.
+2. Clone the forked repository to your local machine.
+3. Create a new branch for your contribution. Choose a descriptive name related to the task you'll be working on.
+4. Make your changes, write your code, and commit them with clear and concise commit messages.
+5. Push your branch to your forked repository on GitHub.
+6. Open a pull request (PR) from your branch to the main repository's branch. If you work on an open issue please mention it.
+7. Provide a clear title and description for your PR, explaining the changes you made.
+8. Our team will review your code, provide feedback, and discuss any necessary changes.
+9. Make the requested changes, if applicable, and push the updates to your branch.
+10. Once your PR is approved, it will be merged into the main repository.
+
+## Code Guidelines
+
+- Use the flake8 linter to check your code for errors.
+- Try to keep your code easy to understand.
+- Write unit tests for new code or modify existing tests to maintain test coverage.
+- Run the existing tests and ensure they pass before submitting your contribution.
+- Code which not pass flake8 or the test will not be reviewed.
```

Comment:
How about creating the [PULL_REQUEST_TEMPLATE](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/creating-a-pull-request-template-for-your-repository) and creating some checkboxes like "Flake Passes", "Add New Test Cases", "Fix Broken Tests", ...?